features/exception_handling/developer_is_notified_which_example_caused_a_fatal_error.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Developer is notified of which scenario caused a fatal error
+  As a Developer
+  I want to know in which scenario or example my script was running
+  So that I can better trace where my changes caused a fatal error
+
+  @isolated
+  Scenario: Spec attempts to call an undeclared function
+    Given the spec file "spec/Message/Fatal/FatalSpec.php" contains:
+      """
+      <?php
+
+      namespace spec\Message\Fatal;
+
+      use PhpSpec\ObjectBehavior;
+      use Prophecy\Argument;
+
+      class FatalSpec extends ObjectBehavior
+      {
+          function it_fatals_when_calling_an_undeclared_function()
+          {
+              anything();
+          }
+      }
+
+      """
+    And the class file "src/Message/Fatal/Fatal.php" contains:
+      """
+      <?php
+
+      namespace Message\Fatal;
+
+      class Fatal
+      {
+          public function __construct($param)
+          {
+              if ($param == 'throw') {
+                  throw new \Exception();
+              }
+          }
+      }
+
+      """
+    When I run phpspec
+    Then I should see "Fatal error happened while executing the following"
+    And  I should see "it fatals when calling an undeclared function"
+
+    @isolated
+	Scenario: Spec attempts to call an undeclared function with a differenet formatter
```

Comment:
The scenario title should reflect the outcome and reason, e.g. something like 'Fatal error is not shown when formatter does not support it'